### PR TITLE
Create IMAGE_GUIDELINES.md

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @KrauseFx</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane/assets/IMAGE_LICENSE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Google open source projects may contain logos, trademarks or service names. These are "Brand Features", and if you want to use them, please do so consistent with [google.com/permissions/trademark/rules.html](https://www.google.com/permissions/trademark/rules.html).
```
Comment (alexsinger) [inline fastlane/assets/IMAGE_LICENSE.md]: I worry legal will have issues with using the word LICENSE in the file name. What about IMAGE_GUIDELINES?

Comment (alexsinger) [inline fastlane/assets/IMAGE_LICENSE.md]: Or IMAGE_USAGE_GUIDELINES

Comment (KrauseFx) [inline fastlane/assets/IMAGE_LICENSE.md]: Great pobnt

**Comment by alexsinger** [comment]
👍

**Review by alexsinger — APPROVED** [review]

**Review by mfurtak — APPROVED** [review]

**Pull request merged**